snmp/tests/compose/data/cisco-5500-wlc.snmprec

```diff
@@ -0,0 +1,556 @@
+1.3.6.1.2.1.1.1.0|4|Cisco Controller
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.9.1.1069
+1.3.6.1.2.1.1.5.0|4|DVA00WLC01
+1.3.6.1.2.1.1.6.0|4|Datadog Paris
+1.3.6.1.2.1.2.1.0|2|9
+1.3.6.1.2.1.2.2.1.2.1|4|Unit: 0 Slot: 0 Port: 1 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.2|4|Unit: 0 Slot: 0 Port: 2 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.3|4|Unit: 0 Slot: 0 Port: 3 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.4|4|Unit: 0 Slot: 0 Port: 4 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.5|4|Unit: 0 Slot: 0 Port: 5 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.6|4|Unit: 0 Slot: 0 Port: 6 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.7|4|Unit: 0 Slot: 0 Port: 7 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.8|4|Unit: 0 Slot: 0 Port: 8 Gigabit - Level 0x6070001
+1.3.6.1.2.1.2.2.1.2.9|4|Virtual Interface
+1.3.6.1.2.1.2.2.1.5.1|66|1000000000
+1.3.6.1.2.1.2.2.1.5.2|66|1000000000
+1.3.6.1.2.1.2.2.1.5.3|66|10000000
+1.3.6.1.2.1.2.2.1.5.4|66|10000000
+1.3.6.1.2.1.2.2.1.5.5|66|10000000
+1.3.6.1.2.1.2.2.1.5.6|66|10000000
+1.3.6.1.2.1.2.2.1.5.7|66|10000000
+1.3.6.1.2.1.2.2.1.5.8|66|10000000
+1.3.6.1.2.1.2.2.1.5.9|66|1000000000
+1.3.6.1.2.1.2.2.1.6.1|4|  00 F2 8B 97 C8 44
+1.3.6.1.2.1.2.2.1.6.2|4|  00 F2 8B 97 C8 45
+1.3.6.1.2.1.2.2.1.6.3|4|  00 F2 8B 97 C8 46
+1.3.6.1.2.1.2.2.1.6.4|4|  00 F2 8B 97 C8 47
+1.3.6.1.2.1.2.2.1.6.5|4|  00 F2 8B 97 C8 48
+1.3.6.1.2.1.2.2.1.6.6|4|  00 F2 8B 97 C8 49
+1.3.6.1.2.1.2.2.1.6.7|4|  00 F2 8B 97 C8 4A
+1.3.6.1.2.1.2.2.1.6.8|4|  00 F2 8B 97 C8 4B
+1.3.6.1.2.1.2.2.1.6.9|4|  00 F2 8B 97 C8 44
```

Comment:
Double checking if this is the real user data ?
We avoid to publish specific information such as mac adresses, interface names, etc..
You can replace this with dummy values